### PR TITLE
[onert/test] Split GenModelTests.cc

### PR DESCRIPTION
**Pull request by hseok-oh**
- Header file for GenModelTest class
- Divide test for each operation

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

part of #3896

### CODE REVIEW

**Comment by wateret** [comment]
What about changing the folder name`ops` to something like `one_op_tests`? `ops` could be confusing as other types of tests also exist.

File: tests/nnfw_api/src/GenModelTests.h [hunk 1]
```diff
@@ -0,0 +1,108 @@
+/*
```
Comment (wateret) [inline tests/nnfw_api/src/GenModelTests.h:1]: As the class name is `GenModelTest` so what about renaming this file to `GenModelTest.h`?

**Comment by hseok-oh** [comment]
@nnfw-bot test cross-runtime-test-debug

**Comment by hseok-oh** [comment]
@nnfw-bot test cross-runtime-test-debug

**Pull request merged**